draft-ietf-rtcweb-jsep.xml

```diff
@@ -1980,6 +1980,10 @@ candidate:1 1 UDP 1694498815 192.0.2.33 10000 typ host
               <xref target="I-D.ietf-rtcweb-video"></xref>(see Section
               5).</t>
 
+              <t>If codec preferences have been set for the associated
+              transceiver, media formats MUST be generated in the
+              corresponding order.</t>
+
```

Comment:
I think it will be a very confusing API if this happens to a developer:

1.  Set the codec preferences.  Receive a call.  Codec preferences ignored.
2.  Set the codec preferences again (or mid-call).  Re-offer.  Codec preferences ignored.
3.  Learn to remove all but my favorite codec.  It works!  Until it doesn't.
4.  Receive a call from an endpoint that doesn't support my favorite codec.  No video.
5.  Learn to check RtpTransceiver.sender.getParameters().codecs first, then pick my favorite out of that group and then exclude the rest.  It works!

Still better than SDP munging?